modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -239,8 +236,6 @@ RETURN similarTitle.text AS title, score
 
 |===
 
-The results are expected, with papers discussing graph-based nearest-neighbor searches.
```

Comment:
Though perhaps in need of a reword or thought; it would be nice to still point out that the results indeed are in a similar "space" to query vector. Thus the vector search (k-ANN) is working correctly as intended.

Reply:
Ok I will add it back in for now, and think about a rewording later :)